web-app/js/portal/search/FacetMapPanel.js

```diff
@@ -13,7 +13,11 @@ Portal.search.FacetMapPanel = Ext.extend(Portal.common.MapPanel, {
 
     constructor: function (cfg) {
 
-        var layerStore = new Portal.data.LayerStore();
+        var layerStore = new GeoExt.data.LayerStore({
+            layers: [
+                Portal.search.FacetedSearchResultsMiniMap.prototype._getBaseLayer()
```

Comment:
Is there maybe somewhere outside of FacetMapPanel and FacetedSearchResultsMiniMap where we could have this function and they could both use it from there? I feel like picking one or the other to house that method is a bit arbitrary.